index.js

```diff
@@ -93,6 +93,27 @@ function checkTopicQueryParameter(req, res, next) {
     next();
 }
 
+
```

Comment:
Please format the code:
- Remove extra empty lines
- Add empty line after checkRetainFlag function